src/app/(authenticated)/[username]/page.tsx

```diff
@@ -12,8 +16,46 @@ export default async function page({ params }: PageProps) {
   return (
     <Container>
       <div className="flex">
-        <Avatar avatarUrl={userResult.value.avatar}></Avatar>
-        <h1> {userResult.value.displayName}</h1>
+        <div className="w-1/4">
+          <Avatar avatarUrl={userResult.value.avatar} size={150}></Avatar>
```

Comment:
If the component doesn't have any children, it's better to make it self closing (`<Avatar />`)